cluster-up/cluster/kind-k8s-1.17/provider.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+export CLUSTER_NAME="kind-1.17"
```

Comment:
All the other kind providers use X.Y.Z naming. I think we should stay aligned, at least in this PR

Reply:
We are transitioning to major.minor providers new ones has to follow that also is better for CI since you don't have to change project-infra if you upgrade bugfix version at k8s.

Reply:
Ack, but I hate the inconsistency in other kinds. Will not fight on this PR though.